Microsoft.Toolkit.Uwp.UI/Helpers/ThemeListener.cs

```diff
@@ -78,19 +87,25 @@ private void Accessible_HighContrastChanged(AccessibilitySettings sender, object
         private async void Settings_ColorValuesChanged(UISettings sender, object args)
         {
             // Getting called off thread, so we need to dispatch to request value.
-            await CoreApplication.MainView?.CoreWindow?.Dispatcher?.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
-            {
-                // TODO: This doesn't stop the multiple calls if we're in our faked 'White' HighContrast Mode below.
-                if (CurrentTheme != Application.Current.RequestedTheme ||
-                    IsHighContrast != _accessible.HighContrast)
+            await OnColorValuesChanged();
+        }
+
+        internal async Task OnColorValuesChanged()
```

Comment:
>async [](start = 17, length = 5)

No need for this to be async and to await the ExecuteOnUIThreadAsync - it can just return the Task from ExecuteOnUIThreadAsync which will be more efficient.

Reply:
Done.